### PR TITLE
Commerce-10366 Integration tests for Wishlist API

### PR DESCRIPTION
**Pull request by smottal**
https://issues.liferay.com/browse/COMMERCE-10366

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/WishListResourceTest.java [hunk 1]
```diff
@@ -15,14 +15,151 @@
...
+	}
+
+	@Override
+	@Test
+	public void testPatchChannelWishList() throws Exception {
+		WishList postWishList = testPatchChannelWishList_addWishList();
+
+		WishList randomPatchWishList = randomPatchWishList();
+
+		@SuppressWarnings("PMD.UnusedLocalVariable")
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/WishListResourceTest.java:78]: why do you need this annotation if you are using the variable few lines below?

**Comment by smottal** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**